<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticesForm</class>
 <widget class="QWidget" name="NoticesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headerLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notices</string>
   </property>
  </widget>
  <widget class="QWidget" name="Notices" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>590</height>
    </rect>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>111</y>
      <width>541</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>531</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="noticeTxt"/>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>Notices</zorder>
  <zorder>menuButton</zorder>
  <zorder>headerLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
